neo/test/rawiotest/test_spikeglxrawio.py

```diff
@@ -14,7 +14,8 @@ class TestSpikeGLXRawIO(BaseTestRawIO, unittest.TestCase):
         'spikeglx'
     ]
     entities_to_test = [
-        'spikeglx/Noise4Sam_g0'
+        'spikeglx/Noise4Sam_g0',
+        'spikeglx/TEST_20210920_0_g0'
```

Comment:
This should read 
```suggestion
        'spikeglx/Noise4Sam_g0/TEST_20210920_0_g0'
```

Reply:
TEST_20210920_0_g0 file is at the wring place in the gin repo.
It should be at `spikeglx/TEST_20210920_0_g0`
I guess I did the mistake.

Reply:
ok, this should be resolved